main.js

```diff
@@ -0,0 +1,13 @@
+document.querySelector('.menu-bars').addEventListener('click', () => {
+  document.querySelector('#mobile-nav').classList.toggle('invisible');
+});
+
+document.querySelector('#close').addEventListener('click', () => {
+  document.querySelector('#mobile-nav').classList.toggle('invisible');
+});
+
+document.querySelectorAll('.mobile-link').forEach((link) => {
+  link.addEventListener('click', () => {
+    document.querySelector('#mobile-nav').classList.toggle('invisible');
+  });
+});
```

Comment:
- [ ] I think it's better if you stored all selected elements from either the `document.querySelector()` or `document.querySelectorAll()` in a variable. This will help your code to become easily readable.

- [ ] I also recommend using methods (like `addEventListner()`) inside of functions. That'll increase readability & makes it easier to manipulate the DOM.